.github/workflows/instascale-release.yml

```diff
@@ -90,3 +112,8 @@ jobs:
         GITHUB_TOKEN: ${{ github.TOKEN }}
       shell: bash
 
+    - name: Delete the PR branch
```

Comment:
This can be done right after `Wait for PR to merge` to make sure the branch is deleted even if the release action fails for some reason.

Reply:
Done